tests/test-cases/Group1-Docker-Commands/1-23-Docker-Inspect.robot

```diff
@@ -155,3 +155,11 @@ Docker inspect container status
     # keyword at top of file
     ${stopped}=  Get container inspect status  ${container}
     Should Contain  ${stopped}  exited
+
+Docker inspect for macaddress
+    ${rc}  ${container}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name=test-macaddress -d busybox sleep 600
+    Should Be Equal As Integers  ${rc}  0
+    Sleep  60s
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} inspect -f '{{range .NetworkSettings.Networks}}{{.MacAddress}}{{end}}' ${container}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  :
```

Comment:
Maybe `Should Match Regexp` would let you make a more meaningful assertion.

Reply:
Let me try, here we should match macaddress